datafusion/core/src/datasource/file_format/csv.rs

```diff
@@ -1256,4 +1256,181 @@ mod tests {
             .build_decoder();
         DecoderDeserializer::new(CsvDecoder::new(decoder))
     }
+
+    fn csv_deserializer_with_truncated(
```

Comment:
Our csv truncated_rows support, will be included in DF 51.0.0, but not for DF 50.0.0.
https://github.com/apache/datafusion/pull/17465